utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dnsmasq.lua

```diff
@@ -0,0 +1,12 @@
+local ubus = require "ubus"
+
+local function scrape()
+  local u = ubus.connect()
+  local metrics = u:call("dnsmasq", "metrics", {})
+
+  for name, value in pairs(metrics) do
+    metric("dnsmasq_"..name, "counter", nil, value)
```

Comment:
Are we sure dnsmasq metrics are only counters ?

Reply:
Reading dnsmasq code all usages are of the form `metrics[metric_name]++`. They were introduced from this: https://github.com/digineo/dnsmasq/compare/metrics

I would assume they will all remain counters, and add special cases later if dnsmasq adds other non-counter metrics.

Reply:
Perfect, thanks for checking